app/models/accounting/quickbooks/connection.rb

```diff
@@ -50,6 +50,17 @@ def realm_id
         data[:realm_id] unless data.blank?
       end
 
+      def last_updated_at
+        DateTime.parse(data[:last_updated_at]) unless data.blank? || data[:last_updated_at].blank?
```

Comment:
Update: I see you're using .utc in other places with quickbooks data. I think we need to pair on this to get it straight. I don't want this to get ugly. Can you please reach out to me.